x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -82,6 +101,38 @@ export class TelemetryEventsSender {
     }
   }
 
+  public async fetchDiagnosticAlerts() {
+    const query = {
+      expand_wildcards: 'open,hidden',
+      index: 'logs-endpoint.diagnostic.collection-default*',
+      ignore_unavailable: true,
+      size: this.maxQueueSize,
+      body: {
+        query: {
+          range: {
+            'event.ingested': {
+              gte: 'now-5m',
+              lt: 'now',
+            },
+          },
+        },
+        sort: [
+          {
+            'event.ingested': {
+              order: 'asc',
```

Comment:
Do you want `asc` here so we get the most recent events?

Reply:
I got it wrong. Updated here https://github.com/elastic/kibana/pull/84422/commits/5638da9c7ddbd3e952cfa65baf24f947a3613c4e